terraform/modules/iam_role_policy/cloudwatch/policy.json

```diff
@@ -0,0 +1,18 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": [
+        "logs:CreateLogGroup",
+        "logs:CreateLogStream",
+        "logs:PutLogEvents",
+        "logs:DescribeLogStreams",
+        "cloudwatch:ListMetrics",
```

Comment:
Hm, WDYT about writing a separate story for logs / metrics permissions and handling it separately from creds -> profiles? Or is that what you're getting at?